amundsen_application/static/js/components/common/ResourceList/index.tsx

```diff
@@ -49,40 +58,71 @@ class ResourceList extends React.Component<ResourceListProps, ResourceListState>
     }
   };
 
+  onViewAllToggle = () => {
+    this.setState({ isExpanded: !this.state.isExpanded })
+  };
 
   render() {
-    const { allItems, slicedItems, itemsPerPage, paginate, source } = this.props;
+    /* TODO ttannis: create render helpers */
```

Comment:
Separate components. Will help fix a bug in `MyBookmarks`.

Reply:
Will fix on master after merging to keep this PR dashboard specific. The bug in question an existing bug from master and not dashboard specific. Ticket is already in progress for this sprint.